maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -1568,13 +1572,13 @@ private void convertGroupParameters()
 
     private void convertJunitEngineParameters()
     {
-        if ( this.getJunitIncludeEngine() != null )
+        if ( StringUtils.isNotBlank( getJunitIncludeEngine()) )
```

Comment:
After you removed the Line 65, you can simply type:
`if ( isNotBlank( getJunitIncludeEngine() ) )`
notice that I added a white space between the end-brackets to confirm the checkstyle rule.